CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
-cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR)
 
 option(IRODS_ENABLE_430_COMPATIBILITY "Enables compatibility with iRODS 4.3.0." YES) # Default to YES for community.
+option(IRODS_ENABLE_42X_COMPATIBILITY "Enables compatibility with iRODS 4.2.X." NO)  # Default to NO for community.
```

Comment:
What does it mean for both options to be `NO`?

Reply:
If both are `NO`, then you get 4.3.1+ compatibility.

Reply:
Okay, that makes sense. Had to get my head around 4.3.0 compatibility coming at the exclusion of all future versions. Thanks